packages/atlas/src/views/viewer/NftPurchaseView/NftPurchaseView.styles.ts

```diff
@@ -0,0 +1,239 @@
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
+
+import { ActionBar } from '@/components/ActionBar'
+import { Avatar } from '@/components/Avatar'
+import { Text } from '@/components/Text'
+import { JoyTokenIcon } from '@/components/_icons/JoyTokenIcon'
+import { cVar, media, sizes, zIndex } from '@/styles'
+
+const ACTION_BAR_HEIGHT = '80px'
```

Comment:
This is measured in all the other places in the app, best to do the same here IMO